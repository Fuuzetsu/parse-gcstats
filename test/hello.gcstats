./Hello +RTS -t/tmp/hello.gcstats --machine-readable 
 [("bytes allocated", "51248")
 ,("num_GCs", "1")
 ,("average_bytes_used", "44408")
 ,("max_bytes_used", "44408")
 ,("num_byte_usage_samples", "1")
 ,("peak_megabytes_allocated", "2")
 ,("init_cpu_seconds", "0.000085")
 ,("init_wall_seconds", "0.000082")
 ,("mut_cpu_seconds", "0.000051")
 ,("mut_wall_seconds", "0.000050")
 ,("GC_cpu_seconds", "0.000061")
 ,("GC_wall_seconds", "0.000061")
 ,("exit_cpu_seconds", "0.000008")
 ,("exit_wall_seconds", "0.000008")
 ,("total_cpu_seconds", "0.000206")
 ,("total_wall_seconds", "0.000202")
 ,("major_gcs", "1")
 ,("allocated_bytes", "51248")
 ,("max_live_bytes", "44408")
 ,("max_large_objects_bytes", "41120")
 ,("max_compact_bytes", "0")
 ,("max_slop_bytes", "25224")
 ,("max_mem_in_use_bytes", "2097152")
 ,("cumulative_live_bytes", "44408")
 ,("copied_bytes", "3312")
 ,("par_copied_bytes", "0")
 ,("cumulative_par_max_copied_bytes", "0")
 ,("cumulative_par_balanced_copied_bytes", "0")
 ,("gc_cpu_percent", "0.000000")
 ,("gc_wall_percent", "0.000000")
 ,("fragmentation_bytes", "0")
 ,("alloc_rate", "1014269598")
 ,("productivity_cpu_percent", "0.248412")
 ,("productivity_wall_percent", "0.252121")
 ,("gen_0_collections", "0")
 ,("gen_0_par_collections", "0")
 ,("gen_0_cpu_seconds", "0.000000")
 ,("gen_0_wall_seconds", "0.000000")
 ,("gen_0_max_pause_seconds", "0.000000")
 ,("gen_0_avg_pause_seconds", "0.000000")
 ,("gen_1_collections", "1")
 ,("gen_1_par_collections", "0")
 ,("gen_1_cpu_seconds", "0.000061")
 ,("gen_1_wall_seconds", "0.000061")
 ,("gen_1_max_pause_seconds", "0.000061")
 ,("gen_1_avg_pause_seconds", "0.000061")
 ]
