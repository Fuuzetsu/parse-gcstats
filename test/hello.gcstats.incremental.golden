bytes allocated: 51248.0
num_GCs: 1.0
average_bytes_used: 44408.0
max_bytes_used: 44408.0
num_byte_usage_samples: 1.0
peak_megabytes_allocated: 2.0
init_cpu_seconds: 8.5e-5
init_wall_seconds: 8.2e-5
mut_cpu_seconds: 5.1e-5
mut_wall_seconds: 5.0e-5
GC_cpu_seconds: 6.1e-5
GC_wall_seconds: 6.1e-5
exit_cpu_seconds: 8.0e-6
exit_wall_seconds: 8.0e-6
total_cpu_seconds: 2.06e-4
total_wall_seconds: 2.02e-4
major_gcs: 1.0
allocated_bytes: 51248.0
max_live_bytes: 44408.0
max_large_objects_bytes: 41120.0
max_compact_bytes: 0.0
max_slop_bytes: 25224.0
max_mem_in_use_bytes: 2097152.0
cumulative_live_bytes: 44408.0
copied_bytes: 3312.0
par_copied_bytes: 0.0
cumulative_par_max_copied_bytes: 0.0
cumulative_par_balanced_copied_bytes: 0.0
gc_cpu_percent: 0.0
gc_wall_percent: 0.0
fragmentation_bytes: 0.0
alloc_rate: 1.014269598e9
productivity_cpu_percent: 0.248412
productivity_wall_percent: 0.252121
gen_0_collections: 0.0
gen_0_par_collections: 0.0
gen_0_cpu_seconds: 0.0
gen_0_wall_seconds: 0.0
gen_0_max_pause_seconds: 0.0
gen_0_avg_pause_seconds: 0.0
gen_1_collections: 1.0
gen_1_par_collections: 0.0
gen_1_cpu_seconds: 6.1e-5
gen_1_wall_seconds: 6.1e-5
gen_1_max_pause_seconds: 6.1e-5
gen_1_avg_pause_seconds: 6.1e-5
